README.md

```diff
@@ -179,35 +180,36 @@ Always assigned to a host and is ghosted from the start.<br>
 Team : Impostors<br>
 Basis : Impostor<br>
 
-If the BountyHunters kill their designated target, their next kill cooldown will be much less than usual.<br>
-Killing a player except their current target results in an increased kill cooldown.<br>
+If the BountyHunter kill their designated target, their next kill cooldown will be much less than usual.<br>
+Killing a player that isn't their current target results in an increased kill cooldown.<br>
 The target swaps after a configurable amount of time.<br>
 
 #### Game Options
 
 | Name                                     |
 | ---------------------------------------- |
 | Time To Swap Bounty(s)                   |
-| Kill Cooldown After Killing Bounty(s)    |
-| Kill Cooldown After Killing Others(s)    |
+| Kill Cooldown After Killing Bountys(s)    |
+| Kill Cooldown After Killing Other(s)     |
 
 ### Camouflager
 
 Team : Impostors<br>
 Basis : Shapeshifter<br>
 
-The Camouflager is an Impostor who can have every player shift into the chosen player for a limited time.
+The Camouflager is an Impostor who can make every player shift into the chosen player for a limited time.<br>
+Nobody can report a corpse while shifted.<br>
 
 ### FireWorks
 
-Create and idea by こう。<br>
+Idea and created by こう。<br>
```

Comment:
something like "Inspiration and creation by [name here]" sounds better, but this could be a personal preference on my end.